source/documentation/monitoring-an-app/how-to-use-cloudwatch-datasource.html.md.erb

```diff
@@ -0,0 +1,52 @@
+---
+title: Using the CloudWatch data source in Grafana 
+last_reviewed_on: 2020-02-14
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+### Introduction
+
+Cloud-platform team added [CloudWatch data source][CloudWatch-datasource] in Grafana, using this [CloudWatch data source][CloudWatch-datasource] you can build dashboards for your CloudWatch metrics.
+
+#### Using the CloudWatch data source in Grafana 
+
+Cloud-platform team created some default dashboards, you can access them [here][cp-dashboards] or you can build your own dashboards for your CloudWatch metrics, you can also do dynamic queries using dimension wildcards.
+
+Create a dashboard using CloudWatch data source:
+
+From the official & community built [dashboards][default-dashboards] for CloudWatch data source, select the AWS resource you want to create the dashboard. Download it as JSON and modify it as you see fit. Copy the JSON string into a `ConfigMap` according to the example below.
```

Comment:
" for CloudWatch data source" - I think you can remove these words, because the whole article is about this.

"select the AWS resource you want to create the dashboard." you need a "for" here, as in "select the AWS resource you want to create the dashboard for."

Strictly speaking the correct way to put it would be; "select the AWS resource for which you want to create the dashboard." - but that's a bit clunky, and the colloquial usage is easier to read.